### PR TITLE
[feat] Add dispatcher service

### PR DESCRIPTION
**Pull request by aquamatthias**
# Description

# To-Dos

- [x] Add test coverage for new or updated functionality
- [x] Lint and test with `nox`

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://some.engineering/code-of-conduct).

### CODE REVIEW

File: fixbackend/dispatcher/next_run_repository.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+from fixbackend.ids import CloudAccountId
+from fixbackend.types import AsyncSessionMaker
+
+
+class NextRun(Base):
+    __tablename__ = "next_run"
+    __table_args__ = (Index("idx_at", "at"),)
+
+    cloud_account_id: Mapped[CloudAccountId] = mapped_column(GUID, primary_key=True)
+    at: Mapped[datetime] = mapped_column(DATETIME, nullable=False)
```
Comment (meln1k) [inline fixbackend/dispatcher/next_run_repository.py:31]: you can also set index=True here.

Comment (aquamatthias) [reply fixbackend/dispatcher/next_run_repository.py]: done

File: migrations/versions/2023-09-28T13:38:59Z_add_next_run_table.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        sa.Column("at", sa.DATETIME(), nullable=False),
+        sa.PrimaryKeyConstraint("cloud_account_id"),
+    )
+    op.create_index("idx_at", "next_run", ["at"], unique=False)
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index("idx_at", table_name="next_run")
```
Comment (meln1k) [inline migrations/versions/2023-09-28T13:38:59Z_add_next_run_table.py:35]: do you want to remove the downgrade action?

File: tests/fixbackend/auth/router_test.py [hunk 3]
```diff
@@ -36,33 +30,18 @@ async def verify(self, user: User, token: str, request: Optional[Request]) -> No
...
-    app.dependency_overrides[get_config] = lambda: default_config
-    app.dependency_overrides[fix_dependencies] = lambda: deps
-
-    async with AsyncClient(app=app, base_url="http://test") as ac:
-        yield ac
-
-
 @pytest.mark.asyncio
-async def test_registration_flow(client: AsyncClient) -> None:
+async def test_registration_flow(api_client: AsyncClient, fast_api: FastAPI) -> None:
```
Comment (meln1k) [inline tests/fixbackend/auth/router_test.py:34]: nice!

**Review by meln1k — COMMENTED** [review]
One comment, otherwise lgtm

File: migrations/versions/2023-09-28T13:38:59Z_add_next_run_table.py (hunk 2 above)
Comment (aquamatthias) [reply migrations/versions/2023-09-28T13:38:59Z_add_next_run_table.py]: done

**Review by meln1k — APPROVED** [review]

**Pull request merged**